Docs/Configuration.tex

```diff
@@ -3288,12 +3287,12 @@ \subsection{Security Properties}\label{miscsecurityprops}
   \href{https://support.apple.com/en-us/HT208330}{\texttt{Full Security}} of Apple
   Secure Boot.
 
-  To start using personalised Apple Secure Boot one will have to reinstall the
-  operating system or personalise it. Until the operating system is personalised
-  one will only be able to load macOS DMG recovery. If DMG recovery is missing,
+  To start using personalised Apple Secure Boot, the  operating system will have
```

Comment:
Please remove the extra space between *the* and *operating*.

Reply:
Noted